geometry_msgs/Point centroid
geometry_msgs/Polygon axis
